src/vivarium_public_health/risks/base_risk.py

```diff
@@ -86,22 +86,19 @@ class Risk(Component):
     # Properties #
     ##############
 
+    @property
+    def name(self) -> str:
+        return self.risk
+
     @property
     def configuration_defaults(self) -> Dict[str, Any]:
         return {
-            self.risk.name: {
-                "exposure": self.build_lookup_table_config(
-                    value="data",
-                    continuous_columns=["age", "year"],
-                    categorical_columns=["sex"],
-                    key_name=f"risk_factor.{self.risk.name}.exposure",
-                ),
-                "ensemble_distribution_weights": {
-                    "key_name": f"risk_factor.{self.risk.name}.exposure_distribution_weights",
-                },
-                "exposure_standard_deviation": {
-                    "key_name": f"risk_factor.{self.risk.name}.exposure_standard_deviation",
+            self.name: {
+                "data_sources": {
+                    "exposure": f"{self.risk}.exposure",
                 },
+                "ensemble_distribution_weights": f"{self.risk}.exposure_distribution_weights",
```

Comment:
Shouldn't these two be in the data sources?

Reply:
Oh, good catch. It doesn't make a difference as it turns out, since we don't iterate through these keys, but they should be there for consistency.